www/js/lib/uiUtil.js

```diff
@@ -415,9 +415,12 @@ define(rqDef, function() {
      * For each contentTheme, a stylesheet must be provided in www/css that is named 'kiwixJS' + contentTheme
      * A rule may additionally be needed in app.css for full implementation of contentTheme
      * 
-     * @param {String} theme The theme to apply (light|dark[_invert|_mwInvert])
+     * @param {String} theme The theme to apply (light|dark[_invert|_mwInvert]|auto[_invert|_mwInvert])
      */
     function applyAppTheme(theme) {
+        var darkPreference = window.matchMedia('(prefers-color-scheme:dark)');
+        var selectedThemeContent = theme.replace(/^[^_]*/, '');
+        if (/^auto/.test(theme)) theme = darkPreference.matches ? ("dark"+selectedThemeContent) : ("light"+selectedThemeContent);
```

Comment:
Minor, but please make use of quote marks and spacing match the style of this block in above line:

`('dark' + selectedThemeContent) : ('light' + selectedThemeContent)`

I'm slightly confused about the last option here `('light' + selectedThemeContent)`. This could produce a theme name like `light_mwInvert`, which is not a currently valid theme name. Wouldn't it be better for this last option just to be `'light'`?

Reply:
@Maneet-k You haven't addressed my specific question here. You are producing options that are invalid, it seems to me. This code can produce a theme `light_mwInvert` which makes no sense to me.